core/templates/dev/head/domain/dashboard/ShowExplorationTitleSection.js

```diff
@@ -0,0 +1,17 @@
+oppia.directive('explorationTitleSection', function($compile) {
+  return {
+    restrict: 'E',
+    scope: {
+      exploration: '=',
+      activeExplorationId: '='
```

Comment:
I have done it for explorations. But when I tried the same for activeExplorationId it didn't quite work. There is a problem with the small arrow which appears: 
![image](https://cloud.githubusercontent.com/assets/13691317/19412761/3371a164-930c-11e6-811f-cb2a21ac47c4.png)

When I try something like this:
```
    scope: {
      getExploration: '&exploration',
      getActiveExplorationId: '&activeExplorationId'
    },
    link: function(scope, element) {
      scope.exploration = scope.getExploration();
      scope.activeExplorationId = scope.getActiveExplorationId();
```
the arrow does not appear:
![image](https://cloud.githubusercontent.com/assets/13691317/19412791/cb6fd684-930c-11e6-8baf-a54fe5d9d85d.png)